### PR TITLE
chore: init readme

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

init readme with minimized content.

## Linear Issue Reference

LOG-2155

## Testing
[preview](https://github.com/logto-io/swift/tree/gao-log-2155-readme)

## Reviewers

@logto-io/eng

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+The monorepo for SDKs and social plugins written in Swift. Check out our [integration guide](https://docs.logto.io/integrate-sdk/swift) or [docs](https://docs.logto.io/sdk/swift) for more information.
+
+We also provide [集成指南](https://docs.logto.io/zh-cn/integrate-sdk/swift) and [文档](https://docs.logto.io/zh-cn/sdk/swift) in Simplified Chinese.
+
+## Installation
+
+### Swift Package Manager
+
+Use the following URL to add Logto SDKs as a dependency:
```
Comment (IceHe) [inline README.md:19]: Use the following URL to add Logto SDKs as **dependencies**?

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+## Products
+
+| Name | Description |
+|---|---|
+| Logto | Logto swift core. |
+| LogtoClient | Logto swift client. |
+| LogtoSocialPlugin | Social plugin foundation for LogtoClient. |
+| LogtoSocialPluginWeb | Social plugin for OAuth-like web IdPs. |
+| LogtoSocialPluginAlipay | Social plugin for Alipay sign in. |
+| LogtoSocialPluginWechat | Social plugin for Wechat sign in. |
```
Comment (IceHe) [inline README.md:34]: `WeChat`
image

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+```
+
+## Products
+
+| Name | Description |
+|---|---|
+| Logto | Logto swift core. |
+| LogtoClient | Logto swift client. |
+| LogtoSocialPlugin | Social plugin foundation for LogtoClient. |
+| LogtoSocialPluginWeb | Social plugin for OAuth-like web IdPs. |
```
Comment (IceHe) [inline README.md:32]: `IDPs`
image

File: README.md (hunk 1 above)
Comment (gao-sun) [reply README.md:19]: it's a single dependency with multiple products in SPM

File: README.md (hunk 2 above)
Comment (gao-sun) [reply README.md:34]: we treat Wechat as a whole across our code. Like `Oidc`

File: README.md (hunk 3 above)
Comment (gao-sun) [reply README.md:32]: it's Q&A not spec, also we are pointing OAuth Identity Provider, which makes sense using `IdP`.

you can search over the internet, just this page uses `IDP`.

**Review by wangsijie — APPROVED** [review]

**Review by IceHe — APPROVED** [review]
LGTM

**Pull request merged**